README.md

```diff
@@ -139,3 +139,21 @@ When you will open your fist Python file, you will be prompted by PyDev to defin
 You can validate the dialog by clicking the OK button.
 
 You can refere to the plugin documentation for usage reference. The documentation is accessible by using the Help > Help contents menu from capella, then on the right tree, you can select Python4Capella.
+
+# Errors in provided scripts
+
+If you open provided scripts, you will notice some errors in them. Those errors can be ignored. They are related to the use of EASE that allows access to Java objects with the PyDev editor:
+
+- some import directives from EASE are not recognized by PyDev
+
+![Python4Capella project content](https://raw.githubusercontent.com/labs4capella/python4capella/master/README/error_import.png)
+
+- some methods defined in EASE modules are not recognized by PyDev
```

Comment:
Make "methods" bold and give some more examples

Reply:
done